src/containers/AccountModal/AccountModal.tsx

```diff
@@ -96,9 +101,12 @@ const AccountModal = () => {
     }
   };
 
+  const shouldHideBanner = ['delete-account'].includes(view ?? '');
+  const dialogSize = isConfirmation ? 'large' : 'small';
+
   return (
-    <Dialog open={!!viewParam} onClose={closeHandler}>
-      <div className={styles.banner}>{banner ? <img src={banner} alt="" /> : null}</div>
+    <Dialog size={dialogSize} open={!!viewParam} onClose={closeHandler}>
+      {!shouldHideBanner && banner && <div className={styles.banner}>{banner ? <img src={banner} alt="" /> : null}</div>}
```

Comment:
Looks like we have already checked for a banner here

```suggestion
      {!shouldHideBanner && banner && <div className={styles.banner}><img src={banner} alt="" /></div>}
```